cves/kernel/CVE-2017-18079.yml

```diff
@@ -173,8 +181,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    An automated tool could have been used to detect a null pointer exception
+    caused by this vulnerability.
```

Comment:
Can you go a little more in depth on how an automated tool could catch this?

Reply:
Yeah, I guess that is a bit vague. I'll add an example of a tool to it.